boards/litex/arty/README.md

```diff
@@ -120,8 +120,8 @@ To boot via serial run the LiteX-included `litex_term.py` (sometimes
 available as `lxterm`):
 ```
 $ ./litex/litex/tools/litex_term.py \
-    --speed 10000000 \
-	--serial-boot \
+    --speed 1000000 \
```

Comment:
going from 10MBaud to 1MBaud is a pretty serious slowdown – any risk of buffer overruns here? (said, having had a miserable experience tracking down a transient bug when moving a different project from 3MBaud to 1MBaud to run more reliably on USB stacks within VMs...)

Reply:
I should've specified that more. In this case, the documentation is wrong and has been for a long time. 10MBaud would be nice but I have deemed that to be unreliable on the Arty. The bitstreams have always been built with a 1MBaud UART (by default). This was actually reported to me some time ago by @irina-b-dev.